oaknational/Oak-Web-Application
# [CUR-369] fix: update robots.txt

## Description

- Remove unnecessary `robots.txt` rules
- Update disallow to exclude lesson journey units and lessons

## How to test

1. Go to https://deploy-preview-2163--oak-web-application.netlify.thenational.academy/robots.txt
2. You should see the robots.txt file with the redundant and incorrect lines removed in the first section:
```
# *
User-agent: *
Disallow: /teachers/programmes/
Disallow: /teachers/lessons/
```
